ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/SqlExecutor.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao
+
+import com.daml.logging.LoggingContext
+import com.daml.metrics.DatabaseMetrics
+
+import java.sql.Connection
+import scala.concurrent.Future
+
+private[platform] trait SqlExecutor {
```

Comment:
I like to have an abstraction for the DbDispatcher, but not sure about the naming here.
How about naming it DbDispatcher, and change the implementation?
Also: I would love to see this rolling forward fully in our codebase -- most of the places we want to depend on the abstraction anyway. (if we name it DbDispatcher, no big refactoring needed)
what do you think @rautenrieth-da @tudor-da @mziolekda

Reply:
Let's address this in a follow up PR. Let's create a ticket for it in the ledger metering epic

Reply:
@mziolekda will create.